src/libs/OptionsListUtils.ts

```diff
@@ -1915,7 +1924,7 @@ function getOptions(
         optionsToExclude.push({login});
     });
 
-    let recentReportOptions = [];
+    let recentReportOptions: ReportUtils.OptionData[] = [];
     let personalDetailsOptions: ReportUtils.OptionData[] = [];
 
     if (includeRecentReports) {
```

Comment:
Passing `includeRecentReports === true` and `recentAttendees` will completely ignore `recentAttendees` argument currently, I think we should move `else if (recentAttendees && recentAttendees?.length > 0) {` to here.

Reply:
`includeRecentReports` is false now, it should work fine when BE will be ready

Reply:
I did it on purpose this way cause the cases where we use `includeRecentReports` and `recentAttendees` are mutually exclusive 🤓
Also `includeRecentReports` value is `false` by default, so there's no need to explicitly set it

Reply:
Ultimately I won't block the PR for this, so feel free to do whatever you think makes more sense 😄